model/flow/epoch.go

```diff
@@ -215,14 +225,42 @@ func (er *EpochRecover) EqualTo(other *EpochRecover) bool {
 	return true
 }
 
-// EpochCommit is a service event emitted when epoch setup has been completed.
-// When an EpochCommit event is emitted, the network is ready to transition to
-// the epoch.
+// EpochCommit is a service event emitted when the preparation process for the next epoch is complete.
+// EpochCommit events must:
+//   - be emitted exactly once per epoch after the corresponding EpochSetup event
+//   - be emitted prior to the epoch commitment deadline (defined by EpochCommitSafetyThreshold)
+//
+// If either of the above constraints are not met, the service event will be rejected and Epoch Fallback Mode [EFM] will be triggered.
+//
+// When an EpochCommit event is accepted and incorporated into the Protocol State, this guarantees that
+// the network will proceed through that epoch's defined view range with its defined committee. It also
+// causes the current epoch to enter the EpochPhaseCommitted phase.
 type EpochCommit struct {
-	Counter            uint64              // the number of the epoch
-	ClusterQCs         []ClusterQCVoteData // quorum certificates for each cluster
-	DKGGroupKey        crypto.PublicKey    // group key from DKG
-	DKGParticipantKeys []crypto.PublicKey  // public keys for DKG participants
+	// Counter is the epoch counter of the epoch being committed
+	Counter uint64
+	// ClusterQCs is an ordered list of root quorum certificates, one per cluster.
+	// EpochCommit.ClustersQCs[i] is the QC for EpochSetup.Assignments[0]
+	ClusterQCs []ClusterQCVoteData
+	// DKGGroupKey is the group public key associated with the epoch's DKG.
+	DKGGroupKey crypto.PublicKey
```

Comment:
I know we're using the `DKG` prefix for most of the indices and keys used and output by DKG (it was not introduced by this PR and I'm not asking to rename them, it is too much work), but it would be great to think of them as the random beacon indices and keys, since they are defined and generated for the random beacon, and will be used later by the threshold signature. Developers looking at the code base may find it confusing to find out that the `DKGGroupKey` in this module is actually the same `RandomBeaconGroupKey` in another module. Maybe we could update the comments to reflect that.

A vague analogy is to generate a machine account key, and call it `KeyGenerationKey` instead of `MachineAccountKey`.

Reply:
Good point, updated the docs accordingly in https://github.com/onflow/flow-go/pull/6236/commits/9b6209cc158cc375f9960592af3a54f2230c81d1.